core-java-modules/core-java-security-2/README.md

```diff
@@ -12,4 +12,5 @@ This module contains articles about core Java Security
 - [How to Read PEM File to Get Public and Private Keys](https://www.baeldung.com/java-read-pem-file-keys)
 - [Listing the Available Cipher Algorithms](https://www.baeldung.com/java-list-cipher-algorithms)
 - [Get a List of Trusted Certificates in Java](https://www.baeldung.com/java-list-trusted-certificates)
+- [Java AES encryption and decryption](https://www.baeldung.com/)
```

Comment:
Let's revert the changes to the README.md - The editorial team will take care of this change.